app/views/index.erb

```diff
@@ -0,0 +1,29 @@
+<center>
+
+<br><br>
+<h1>Welcome to Chitter</h1>
+<br>
+
+  <% if current_maker %>
+    <form action="/peeps" method="post" class="new-peep">
```

Comment:
see this article on best practice for html forms: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Forms/How_to_structure_an_HTML_form